03cpp2/sec02SmartPointers.md

```diff
@@ -235,8 +212,14 @@ title: Smart Pointers
 * Always read the manual
 * Always consistently use it
 
+### Homework - 14
+
+* Create a memory leak using bare pointers and `new` to heap allocate and then repeat using using smart pointers 
```

Comment:
A suggestion, could it be interesting to introduce tools like `valgrind` to consistently check for memory leaks ? Or at least suggest it ?

Reply:
Good point, I've added a note that they will learn how to consistently check for memory leaks using `valgrind` in lecture 6